egs/wsj/s5/steps/cleanup/resegment_data_dir.sh

```diff
@@ -0,0 +1,165 @@
+#!/bin/bash
+
+# Modified based on the script: utils/data/copy_data_dir.sh 
+
+# Copyright 2013  Johns Hopkins University (author: Daniel Povey)
+# Apache 2.0
+
+# This script operates on a directory, such as in data/train/,
+# that contains some subset of the following files:
+#  feats.scp
+#  wav.scp
+#  spk2utt
+#  utt2spk
+#  text
+#
+# It copies to another directory, possibly adding a specified prefix or a suffix
+# to the utterance and/or speaker names.  Note, the recording-ids stay the same.
+#
+
+# begin configuration section
+frame_shift=10   # In milliseconds
+validate_opts=   # should rarely be needed.
+# end configuration section
+
+. utils/parse_options.sh
+
+if [ $# != 4 ]; then
+  echo "Usage: "
+  echo "  $0 <srcdir> <sub-segment-text> <sub-segments> <destdir>"
```

Comment:
Please do a better job job explaining this interface in the usage message.  E.g. show the format of sub-segment-text and sub-segments, and explain what the script does.